lib/engine_config.js

```diff
@@ -0,0 +1,48 @@
+var fs = require("fs");
+
+function EngineConfig(path) {
+  this.engineJSON = JSON.parse(fs.readFileSync(path));
+};
+
+EngineConfig.prototype.includePaths = function() {
+  return this.engineJSON.include_paths;
+};
+
+// cc-yaml currently ends up passing bools as stringy equivalents
+function _coerceBool(val) {
```

Comment:
Just because it's private, and putting it on the prototype makes it "private by convention" (since it's technically still exposed to outside modules), as opposed to actually really private.